rtf_vis_tool/src/Components/gtsfm_node_list.js

```diff
@@ -2,48 +2,48 @@
 //imported in DivGraph.js
 
 const gtsfm_node_set = ['absolute Rs',
+                        'absolute Ts',
+                        'Aggregate',
+                        'Bundler Calibrator',
+                        'Bundler Pinhole Cameras',
+                        'Data Association w/ Track Filtering',
+                        'Data Loader + Filter Invalid Edges',
+                        'Dense Mesh Reconstruction',
+                        'Dense Point Cloud',
                         'Descriptors',
+                        'Detector',
                         'DigiCamDB',
-                        'E matrix', 
-                        'relative Rs (2): i2_R_i1', 
+                        'E matrix',
+                        'File Writer',
+                        'Image i', 
                         'Images', 
-                        'Post-Processor', 
-                        'relative Rs: i2_r_i1', 
-                        'LargestConnected Component Extractor', 
-                        'Image Pair Indices (i1,i2)', 
-                        'relative Ts: i2_t_i1',  
-                        'Scene Directory',
-                        'Dense Point Cloud', 
-                        'Verified Correspondence Indices',
+                        'Image Pair Indices (i1,i2)',
                         'Intrinsics',
-                        'File Writer', 
-                        'MVSNet', 
-                        'Triangulation', 
-                        'Optimizer', 
                         'Keypoint Describer',
-                        'SfMData', 
-                        'absolute Ts', 
-                        'Output Directory', 
-                        'Scene Image Directories', 
-                        'Detector',
-                        'Aggregate',
-                        'Bundler Calibrator', 
                         'Keypoints',
-                        'Shonan', 
-                        'Zipped Results for All Scenes', 
-                        'Verifier',
-                        '1d-SfM',
-                        'Bundler Pinhole Cameras', 
-                        'Data Loader + Filter Invalid Edges', 
+                        'LargestConnected Component Extractor',
+                        'MVSNet', 
+                        'Optimizer',
+                        'Output Directory',
+                        'Post-Processor',
+                        'Putative Correspondence Indices',
+                        'Putative Matcher',
+                        'relative Rs: i2_r_i1',
+                        'relative Rs (2): i2_R_i1',    
+                        'relative Ts: i2_t_i1',  
```

Comment:
`i2ti1` is our current notation -- that's my bad for having the wrong thing in the slide picture. I'll update it on my end too